kubernetes/samples/charts/ingress-per-domain/README.md

```diff
@@ -0,0 +1,67 @@
+# Ingress per Domain Chart
+This chart is to deploy an Ingress resource in front of a WebLogic domain cluster. We support tow Ingress types: Traeafik and Voyager.
+
+## Prerequisites
+- The corresponding Ingress controller: Traefik or Voyager, is installed in the Kubernetes.
```

Comment:
The corresponding Ingress controller, Traefik or Voyager, is installed in Kubernetes (or in the Kubernetes ?what).